platform/xbox/functions/math/nexttoward.c

```diff
@@ -1,20 +1,109 @@
+// Adapted from public domain code by Danny Smith <dannysmith@users.sourceforge.net>
+
 #include <math.h>
 #include <assert.h>
 
+
```

Comment:
Was this empty line intentional?

Reply:
Yes, to visually separate the functions from header inclusions.